common/djangoapps/util/model_utils.py

```diff
@@ -156,6 +162,41 @@ def emit_settings_changed_event(user, db_table, changed_fields: Dict[str, Tuple[
     USER_FIELDS_CHANGED.send(sender=None, user=user, table=db_table, changed_fields=changed_fields)
 
 
+def emit_course_completed_tracking_event(user, course_id):
```

Comment:
This function looks good, but `common/djangoapps/utils/model_utils.py` isn't really the right place for it - this file is meant to be about django models in the general sense (like stuff that happens when a new column is added).

If this function is only used by the `certificates` app, I'd suggest putting it somewhere in there - `lms/djangoapps/certificates/utils.py` would be a fine place.